HopsanGUI/LogVariable.cpp

```diff
@@ -755,16 +756,16 @@ SharedVectorVariableT VectorVariable::toFrequencySpectrum(const SharedVectorVari
 
         // FFT is symmetric, so only use first half
         // Also skip f=0, but include n/2 (nyquist)
+        double fs = time.length()/maxt;     //Sampling frequency
```

Comment:
.length() is not available in Qt4
Maybe we should drop Qt4 support for the next release, but lets keep it for this one